tests/cases/case_neveralone_home.yaml

```diff
@@ -0,0 +1,251 @@
+  Environment: Settings
+  Precases:
+    - LoginNeverAloneHelper
+  Aftercases:
+    - TerminateApp
+  Roles:
+    - Role: localiOS
+      App: NeverAlone
+    - Role: desktopChrome
+      Capabilities:
+        chromeOptions:
+          args:
+            - use-file-for-fake-audio-capture=$AND_CMD_pwd$/video_audio_samples/sample3.wav
+            - use-file-for-fake-video-capture=$AND_CMD_pwd$/video_audio_samples/sample_640x360.mjpeg
+      App: desktop
+    - Role: command1
+      App: command
+  Actions:
+    # Login to the Care Platform app with Care Partners account.
+    - Type: case
+      Value: CarePartnerLogin
+    # Click on the Call Personal Care Partner button.
+    - Type: click
+      Strategy: class_chain
+      Id: $PAGE.never_alone_home.call_care_partner$
+    # Verify that the screen updates to show screen asking what kinf of heelp you need:
+    # Medical 
+    # Non-Medical
+    # Cancel Call
+    - Type: wait_for
+      Strategy: class_chain
+      Id: $PAGE.never_alone_call.medical_call_button$
+    - Type: wait_for
+      Strategy: class_chain
+      Id: $PAGE.never_alone_call.non_medical_call_button$
+    - Type: wait_for
+      Strategy: class_chain
+      Id: $PAGE.never_alone_call.cancel_call_button$
+    # Click on the Cancel Call button and verify that Home page is displayed.
+    - Type: click
+      Strategy: class_chain
+      Id: $PAGE.never_alone_call.cancel_call_button$
+    - Type: wait_for
+      Strategy: class_chain
+      Id: $PAGE.never_alone_home.call_care_partner$
+    # Click on the Call Personal Care Partner button again.
+    - Type: click
+      Strategy: class_chain
+      Id: $PAGE.never_alone_home.call_care_partner$
+    # Click on the Non Medical call button.
+    - Type: click
+      Strategy: class_chain
+      Id: $PAGE.never_alone_call.non_medical_call_button$
+    # Verify that screen redirects to the call waiting room.
+    - Type: wait_for
+      Strategy: class_chain
+      Id: $PAGE.never_alone_call.waiting_room_label$
+    - Type: click
+      Strategy: class_chain
+      Id: $PAGE.never_alone_call.video_button$
+
+    ## Care partner steps to take the call.
+    - Type: click
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_home.first_call_of_queue$
+    - Type: click
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_home.answer_call$
+    - Type: click
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.video_btn$
+    - Type: wait_for
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.video_mirror_care_partner$
+    - Type: click
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.full_screen_call_btn$
+    - Type: wait_for
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.verify_video_is_full_screen$
+    - Type: click #workaround
```

Comment:
Please add a explanation of what workaround are you referring to

Reply:
this workaround is until CXTB-253 is solved, i added the ticket number to the comment